cleanlab/dataset.py

```diff
@@ -241,12 +242,15 @@ def _2d_matrix_to_row_column_value_list(matrix):
 
         return [(*i, v) for i, v in np.ndenumerate(matrix)]
 
+    if multi_label:
```

Comment:
delete the parts of the docstring that mention this argument, we no longer want this argument to be documented

Reply:
Resolved Here: https://github.com/cleanlab/cleanlab/pull/657/commits/6a6c51212e54a38456c33bc7ce5743c824024e78